studies/cogsci2023/models/prospect_theory.py

```diff
@@ -183,3 +184,11 @@ def plot_prospect_theory(model=None):
 # import matplotlib.pyplot as plt
 # plt.plot(x_range, y)
 # plt.show()
+
+register_model(
+    "prospect_theory",
+    metadata=prospect_theory_metadata,
+    data=prospect_theory_data,
+    synthetic_experiment_runner=prospect_theory_experiment,
+    name="Prospect Theory",
```

Comment:
Do we need the plotter added here?

Reply:
> Curious would this also work if you just import specific models and not the entire model directory?

Absolutely. You could register single models at a time (and that's probably the best way to refactor it – just add the models one by one), or register your own models if you like, and have them available to anything which uses the model inventory.

I envisage us loading all of the local models into the inventory whenever we load the autora.model submodule, but you could also have a situation where for some pre-registered models which have a lot of data there's a placeholder which then loads the model on the fly from a website, _a la_ scikit-learn.

Reply:
> Do we need the plotter added here?

Good catch! Updated.

Reply:
Great!